srcpkgs/schroot/template

```diff
@@ -41,22 +40,23 @@ make_dirs="
 	/var/lib/schroot/session 0755 root root
 	/var/lib/schroot/mount 0755 root root
 	/etc/schroot/chroot.d 0755 root root"
-hostmakedepends="cmake pkg-config automake libtool gettext xz"
+hostmakedepends="cmake pkg-config automake autoconf libtool gettext xz"
 makedepends="boost-devel pam-devel lockdev-devel libuuid-devel e2fsprogs-devel gettext-devel
  libcppunit-devel"
 short_desc="Allows users to execute commands in different chroots"
-maintainer="Andrea Brancaleoni <miwaxe@gmail.com>"
-license="GPL-3"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="GPL-3.0-only"
 homepage="https://wiki.debian.org/Schroot"
 distfiles="
 ${DEBIAN_SITE}/main/s/${pkgname}/${pkgname}_${version}.orig.tar.xz
-${DEBIAN_SITE}/main/s/${pkgname}/${pkgname}_${version}-${_debian_version}.debian.tar.xz"
+http://security.debian.org/debian-security/pool/main/s/${pkgname}/${pkgname}_${version}-${_debian_version}.debian.tar.xz"
 checksum="
 3ce8dfd9cb97b099e4b6d4ccec421d6cc8c9ef84574681e928a12badb5643d0b
-56bc82fc8ae7f6ca7eef506ccc1dca1211b2c84d83efc50d24670b8bdb9ea8bb"
+7bd4e0c2709979362c86a86c10d2b23d290d26e1a2d301a602e829327f483ec1"
 nocross=yes
 
 pre_configure() {
+	sed -i '/zfs-snapshot-support.patch/d' ../debian/patches/series
```

Comment:
why remove this patch?

Reply:
Debian switch to CMake for `schroot` (see `debian/rules`). Build with cmake instead, otherwise `autotools` can't generate because of:
> error: BUILD_ZFSSNAP does not appear in AM_CONDITIONAL

Reply:
Thanks for your help.

I'll have look. :+1:

Reply:
Hey @sgn,

I've read the `debian/rules` now and changed the template accordingly.